bundles/org.openhab.binding.doorbird/src/main/java/org/openhab/binding/doorbird/internal/handler/DoorbellHandler.java

```diff
@@ -412,6 +416,7 @@ private void startMotionOffJob() {
         if (offDelay == null) {
             return;
         }
+        ScheduledFuture<?> motionOffJob = this.motionOffJob;
```

Comment:
Take care, at line 423, you must now use `this.motionOffJob`

Reply:
Good catch!